google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb

```diff
@@ -173,10 +173,12 @@ def retrieve_snapshot project, subscription, snapshot_name
       next_subs.token.must_be :nil?
     end
 
-    it "should allow creation of a subscription" do
-      subscription = topic.subscribe "#{$topic_prefix}-sub3"
+    it "should allow creation of a subscription with options" do
```

Comment:
Sure. The reason I didn't add a new test is because it's acceptance testing, I thought it wasn't worth the cost.